feature_tests/src/structs.rs

```diff
@@ -51,13 +51,11 @@ pub mod ffi {
             Box::new(Opaque("".into()))
         }
 
-        #[diplomat::attr(supports = constructors, constructor)]
```

Comment:
could you add a lowering check that enforces a single constructor?

Reply:
Out of scope, I think

Reply:
then file an issue please